testdata/go/v3/memcached-operator/config/manifests/kustomization.yaml

```diff
@@ -1,4 +1,27 @@
+# These resources constitute the fully configured set of manifests
+# used to generate the 'manifests/' directory in a bundle.
 resources:
+- bases/memcached-operator.clusterserviceversion.yaml
 - ../default
 - ../samples
 - ../scorecard
+
+# [WEBHOOK] To enable webhooks, uncomment all the sections with [WEBHOOK] prefix.
+# Do NOT uncomment sections with prefix [CERTMANAGER], as OLM does not support cert-manager.
```

Comment:
Question: Have this need already tracked on the OLM side?

Reply:
I think https://github.com/operator-framework/operator-lifecycle-manager/issues/1805 was created for this